scripts/ci/deploy.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$ARGV_ARCHITECTURE" \
     -s "$(pwd)" \
-    -c "make publish-aws-s3"
+    -c "make publish-aws-s3 publish-bintray-debian publish-bintray-redhat"
 else
   ./scripts/build/check-dependency.sh make
-  make publish-aws-s3
+  make publish-aws-s3 publish-bintray-debian publish-bintray-redhat
```

Comment:
Hmmm, looks like this is already causing Appveyor to fail https://ci.appveyor.com/project/resin-io/etcher/build/1.0.5317/job/injturoyu7u2fkw7

But why is Appveyor trying to publish a not-yet-merged PR-build anyway??! :confused: